client.js

```diff
@@ -0,0 +1,143 @@
+const http = require('http');
+const net = require('net');
+let args = process.argv;
+
+/**********************
+    User Arguments
+**********************/
+
+if (!args[2]) {
+  console.log(`
+
+****************
+help / usage
+****************
+
+  * Use the 'node client.js' command with a valid web address to 
+    get the data from that endpoint printed for you.
+
+example: node client.js localhost8080/helium.html
+
+  * Additionally, you can specify 3 options after the web address:
+
+    1) Specify the request method. You can specify which http request 
+       method you would like to use by typing either "GET", "POST", 
+       "PUT", or "DELETE" (case-insensitive). By default, the request will
+       be a GET request
+
+    2) If you want to only get the response header back instead of both the 
+       header and full response body, you can specify with the "-h" flag. By
+       default, You will receive the full header and body in the response.
+
+    3) Lastly, you can specify a port if necessary. This can be done by using
+       the keyword "port" immediately followed by the desired port number. By
+       default, port80 will be used.
+
+    These 3 options can be specified in any order and/or specified completely
+    indepenedent of eachother.
+
+    Below is an example of using all 3 options:
+
+example: node client.js google.com -h GET port80
+
+  `);
+
+} else {
+
+  let URI = args[2].slice(args[2].indexOf('/'));
```

Comment:
Instead of always looking for a `/` you could check to see if it exists first. If it does then slice your string, if not,  you can just use it as is.